.github/workflows/workflow-run-docker-rust-build.yaml

```
@@ -25,6 +25,12 @@ on:
         required: false
         type: boolean
         description: Whether to build additional testing images. If not specified, only the base release images will be built
+      TARGET_REGISTRY:
+        default: remote
```

nit: make default GCP only?
```suggestion
        default: gcp
```